doc/sphinx/source/quickstart/installation.rst

```diff
@@ -168,7 +256,7 @@ code (called ESMValTool if you did not choose a different name) and run
 
     conda env create --name esmvaltool --file environment.yml
 
-This installs the ESMValCore package from conda as a dependency.
+This installs the esmvaltool package from conda as a dependency.
```

Comment:
The step above does not install esmvaltool? That's installed from source a few steps lower

Reply:
A bad replacement...